src/convenience_functions.jl

```diff
@@ -1,15 +1,27 @@
 JuMP.value(X::MPSGEScalarVariable) = JuMP.value(get_variable(X)) #Issue if the model isn't defined
 
-JuMP.fix(X::MPSGEScalarVariable, value::Real) = JuMP.fix(get_variable(X), value; force=true)
+function JuMP.fix(X::MPSGEScalarVariable, value::Real) 
+    M = model(X)
+    if !ismissing(numeraire(M)) && X == numeraire(M)
+        M.numeraire = missing
+    end 
+    JuMP.fix(get_variable(X), value; force=true)
+    set_start_value(X, value)
+end
```

Comment:
Slightly more complicated, but you need to check if you're fixing a previous numeraire, if so then the old numeraire is no longer relevant.

Finally, set the starting value in case you haven't yet solved the model.